ansible/roles/ansible-horizon-plugin/tasks/pre_tasks.yml

```diff
@@ -39,3 +39,8 @@
     msg: "{{HORIZON_PATH}}"
     verbosity: "{{verbosity_level}}"
 
+- name: Install the latest version of findutils
+  dnf:
+    name:
+      - findutils
+    state: latest
```

Comment:
we need to look for an alternative of `find` command.
We can not always be sure that the environment has Internet connectivity.